/bitbots_extrinsic_camera_calibration:
  ros__parameters:
    offset_x: -0.02
    offset_y: 0.0
    offset_z: 0.0

/bitbots_extrinsic_imu_calibration:
  ros__parameters:
    offset_x: 0.06
    offset_y: -0.085
    offset_z: 0.0
